content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Pamiętaj:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> Komponenty sieciowe często udostępniają imperatywne API. Na przykład komponent sieciowy `video` może udostępniac funkcje `play()` i `pause()`. Aby skorzystać z imperatywnego API komponentu sieciowego, możesz potrzebować użyć referencji, aby bezpośrednio nawiązać kontakt z węzłem DOMu. Jeśli korzystasz z komponentów sieciowych innych osób, najlepszym rozwiązanie będzie utworzyć komponent React służący jako nakładka dla komponentu sieciowego.
 >
-> Events emitted by a Web Component may not properly propagate through a React render tree.
-> You will need to manually attach event handlers to handle these events within your React components.
+> Wydarzenia emitowane przez komponent sieciowy mogą nie właściwie propagować się przez drzewo renderowania React.
```

Comment:
```suggestion
> Zdarzenia emitowane przez komponent sieciowy mogą niewłaściwie propagować się przez reactowe drzewo renderowania.
```

Reply:
Zmieniłem, ale `reactowe` napisałem z dużej litery.

Reply:
A to dlaczego? Przecież przymiotniki piszemy małymi literami.

Reply:
Generalnie według mnie raczej lepiej byłoby to zapisywać jako jako rzeczownik w dopełniaczu (`Reacta`/`React`), a nie jako przymiotnik (`Reactowe`/`reactowe`) lepiej było by zapisywać to.

Reply:
Tak by było, gdyby "to coś" należało do Reacta, a nie było z nim jedynie związane.
Zresztą, w Poradni PWN napisali taki komentarz do odmiany słowa "Instagram": https://sjp.pwn.pl/poradnia/haslo/O-co-sie-rozchodzi-z-tymi-przymiotnikami;19231.html